provisioner/clusterpy.go

```diff
@@ -219,16 +220,19 @@ func (p *clusterpyProvisioner) provision(
 	}
 
 	// find the best subnet for each AZ
-	azInfo := selectSubnetIDs(subnets)
+	azInfoLBs := selectSubnetIDs(subnets, subnetELBRoleTagName)
+	azInfoWorkers := selectSubnetIDs(subnets, subnetNodeRoleTagName)
```

Comment:
Here we use "node role" and "workers", the annotation is also `kubernetes.io/role/node` and not `kubernetes.io/role/worker` - should this be consistent and use either "node" or "worker"?

Reply:
Updated the variable to be consistent with the tag name after that was changed.